src/odemis/gui/comp/fastem_roa.py

```diff
@@ -151,12 +151,14 @@ def on_points(self, points):
             else:
                 self.calculate_field_indices()
 
-    def estimate_acquisition_time(self):
+    def estimate_acquisition_time(self, acq_dwell_time: Optional[float] = None):
         """
         Computes the approximate time it will take to run the ROA (megafield) acquisition.
+
+        :param acq_dwell_time: (float or None) The acquisition dwell time.
         :return (0 <= float): The estimated time for the ROA (megafield) acquisition in s.
         """
-        field_time = self._detector.frameDuration.value + 1.5  # there is about 1.5 seconds overhead per field
+        field_time = self._detector.getTotalFieldScanTime(acq_dwell_time) + 1.5  # there is about 1.5 seconds overhead per field
```

Comment:
We should keep in mind that this 1.5s might be too much with our throughput improvements. I would keep it for now, but if it turns out that the estimate is too high, we can tune this.